doc/usage/configuration.rst

```diff
@@ -660,10 +660,11 @@ documentation on :ref:`intl` for details.
    generated by Sphinx will be in that language.  Also, Sphinx will try to
    substitute individual paragraphs from your documents with the translation
    sets obtained from :confval:`locale_dirs`.  Sphinx will search
-   language-specific figures named by `figure_language_filename` and substitute
-   them for original figures.  In the LaTeX builder, a suitable language will
-   be selected as an option for the *Babel* package.  Default is ``None``,
-   which means that no translation will be done.
+   language-specific figures named by `figure.language.filename` 
```

Comment:
Hmm? Why did you change `figure_language_filename` to `figure.language.filename`? This is a configuration variable. So this should not be changed.

Reply:
Oh I mistook this for an example. I did not know it was a config variable.

Reply:
Okay. I fixed it in another PR.